lib/android/didkit/src/main/java/com/spruceid/DIDKit.java

```diff
@@ -0,0 +1 @@
+../../../../../../../java/main/com/spruceid/DIDKit.java
```

Comment:
Are these necessary if the Android app depends on the DIDKit package?

Reply:
I've decided to make both the java package and the android library share the same java files, that exposes the interface from `libdidkit.so`.

Don't know if this answers your question, but the android app dependes on the DIDKit android package, that packages the `libdidkit.so` files and the `.java` files (from the Java lib) into a `AAR`.

Reply:
I meant, if you add DIDKit as a dependency in Gradle, you wouldn't need the symbolic links. But now I see what it's doing, exposing the actual files of the Java library in an Android setting -- which confuses me for why that's needed? I had a brief look and having a AAR would allow us to include "Android resources or manifest files", but we don't do that currently, correct?

Another, possibly related, thing I don't understand is why `didkit/lib/android` has gradle files? It doesn't seem to bundle both the library and the example.

Reply:
The gradle files from `didkit/lib/android` are created by Android Studio, it contains "shared" tasks between the app and the lib